src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/builders/DQGeneratorFormBuilder.java

```diff
@@ -234,6 +260,18 @@ public boolean isRespondentState(CaseData caseData) {
             || state.equals(RESPONDENT_RESPONSE_LANGUAGE_IS_BILINGUAL.fullName());
     }
 
+    private boolean isPastDeadlineState(String state) {
+        return TAKEN_OFFLINE_PAST_APPLICANT_RESPONSE_DEADLINE.fullName().equals(state)
+            || PAST_APPLICANT_RESPONSE_DEADLINE_AWAITING_CAMUNDA.fullName().equals(state)
+            || CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE.fullName().equals(state)
+            || PAST_CLAIM_DISMISSED_DEADLINE_AWAITING_CAMUNDA.fullName().equals(state)
+            || CLAIM_DISMISSED_PAST_CLAIM_NOTIFICATION_DEADLINE.fullName().equals(state)
+            || CLAIM_DISMISSED_HEARING_FEE_DUE_DEADLINE.fullName().equals(state)
```

Comment:
Hearing fee comes very later in the journey but DQs happen earlier.

Reply:
Don't think it would do any harm but the idea was do only add past deadline states and this one indeed is not one of them, I will remove it.